server/graphql/v2/object/StripeConnectedAccount.ts

```diff
@@ -10,6 +10,9 @@ export const GraphQLStripeConnectedAccount = new GraphQLObjectType({
   name: 'StripeConnectedAccount',
   description: 'Stripe connected account properties',
   fields: () => ({
+    username: {
+      type: GraphQLString,
+    },
```

Comment:
What kind of permission model do we have here? Are we relying on `GraphQLStripeConnectedAccount` never being returned if you don't have the permission?

Reply:
Yes, this object is returned by the Host resolver and the permission check is being done there.